<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectItem</class>
 <widget class="QWidget" name="ObjectItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>128</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { 
	background-color :  grey;
 }</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="blurLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="selectFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QFrame" name="validFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <zorder>imageLabel</zorder>
  <zorder>validFrame</zorder>
  <zorder>selectFrame</zorder>
  <zorder>blurLabel</zorder>
  <zorder>typeLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
